src/app/icd/server/ICDManager.h

```diff
@@ -85,12 +85,12 @@ class ICDManager : public ICDListener, public TestEventTriggerHandler
      *         When this event is called, the ICD is still in ActiveMode.
      *        If the ActiveMode timer is increased due to the TransitionToIdle event, the event will not be called a second time in
      *        a given cycle.
-     *        OnEnterIdleMode will always the third when the ICD has transitioned to IdleMode.
+     *        OnEnterIdleMode will always the third event and indictats that the ICD has transitioned to IdleMode.
```

Comment:
```suggestion
     *        OnEnterIdleMode will always the third event and indicates that the ICD has transitioned to IdleMode.
```

Is it true it's always the 3rd call?

Also "OnEnterIdleMode does not exist anywhere.

Reply:
Yes, it is always the 3rd call.
You might have a version before it was used. We use it in our example.